### ISSUE TITLE
System Informer 3.1.24333 (Canary build) & older do not support Windows Kernel 10.0.26100.2605

### ISSUE THREAD

**Issue by chuckchopp** [winsiderss/systeminformer#2330]
### Description of the feature, modification, idea or suggestion

When running systeminformer-3.1.24333-canary-bin\amd64\SystemInformer.exe on Windows 11 24H2 (OS Build 26100.2605) w/the latest Windows Updates applied as of 12/12/2024, I get an error indicating the kernel is not supported.

The error message dialog shows:

"Unable to load kernel driver"

...

Windows 11 24H2 10.0.26100
Windows Kernel 10.0.26100.2605
System Informer 3.1.24333.0 Release

![image](https://github.com/user-attachments/assets/ca263b02-8a27-43c3-b50b-38e107ebd5f2)

I had been running the most recent stable release and got the same error, and had given the canary build a try to see if it already addressed this issue.  Given that the same error is still occurring, it looks like there is an actual need to have the driver updated again for this build of Windows.

### Proposed implementation details (optional)

The kernel driver needs to be updated to support this latest build of Windows 11 24H2.

**Comment by dmex** [comment]
Duplicate of #1823

**Comment by chuckchopp** [comment]
This is not a duplicate of #1823 in terms of the actual System Informer version and the Windows Kernel Version information.  Granted, it's the same type of request in terms of needing to have the System Informer device driver updated for a newer version of the Windows Kernel, but the version information I reported very new as of 12/2024, not something that was released in 2023.

What I reported:

Windows 11 24H2 10.0.26100
Windows Kernel 10.0.26100.2605
System Informer 3.1.24333.0 Release

What is in #1823:

Windows 11 23H2 10.0.22631
Windows Kernel 10.0.2263131.55
System Informer  3.0.7435 (0524888)


This issue is an ongoing active defect in the latest builds of System Informer and it absolutely requires an updated device driver to get it resolve.
